styles.css

```diff
@@ -0,0 +1,458 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+@import url('https://fonts.cdnfonts.com/css/metropolis-2');
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Metropolis', sans-serif;
+    color: #000;
+}
+
+header {
+    height: 75px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-bottom: 1px solid #ebf2f2;
+}
+
+header img {
+    height: 46px;
+}
+
+nav {
+    display: flex;
+    align-items: center;
+}
+
+.menu {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    justify-content: end;
+    width: 1044px;
+}
+
+.menu li {
+    position: relative;
+    background-color: #fff;
+}
+
+.menu li a {
+    text-decoration: none;
+    padding: 8px 12px;
+    font-size: 15.2px;
+    display: block;
+    align-self: center;
+    font-weight: 700;
+}
+
+.menu li a::after {
+    content: "";
+    display: inline-block;
+    width: 15px;
+    height: 15px;
+    margin-left: 5px;
+    background-image: url('./images/down.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+    vertical-align: middle;
+}
+
+.dropdown {
+    display: none;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    list-style: none;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    z-index: 1;
+    font-size: 1rem;
+    padding: 1rem;
+    background-color: #fff;
+    white-space: nowrap;
+}
+
+.menu-icon {
+    display: none;
+}
+
+.menu li:hover .dropdown {
+    display: block;
+}
+
+.dropdown li {
+    padding: 10px;
+}
+
+.dropdown li a.link-secondary {
+    color: #086dc3;
+}
+
+.dropdown li a {
+    display: block;
+    padding: 0;
+    text-decoration: none;
+    color: black;
+    font-weight: 400;
```

Comment:
`400` is a default value. Please avoid adding lines that add 0 value to the codebase.